nntrainer/tensor/tensor.cpp

```diff
@@ -750,6 +752,49 @@ Tensor Tensor::getSharedDataTensor(const TensorDim dim_, unsigned int offset,
   return ret;
 }
 
+std::vector<Tensor> Tensor::split(unsigned num_size, int axis) {
+  if (axis == -1) {
+    axis = 3;
+  }
+  NNTR_THROW_IF(!(0 <= axis && axis < 4), std::invalid_argument)
```

Comment:
add a throw for strided tensors as well.

Reply:
doesn't apply support strided tensors? if not I'll make a throw here

Reply:
yup, my bad, it will work.